src/riverroute/mosart_restfile.F90

```diff
@@ -248,7 +248,7 @@ subroutine restFile_read_pfile( pnamer )
 
       if (mainproc) then
          call get_curr_date(yr, mon, day, tod)
-         write(timestamp,'(".",i4.4,"-",i2.2,"-",i2.2,"-",i5.5)'),yr,mon,day,tod
+         write(timestamp,'(".",i4.4,"-",i2.2,"-",i2.2,"-",i5.5)') yr,mon,day,tod
```

Comment:
I assume this is the bug-fix. The rest changes the ChangeLog.

Approving.

@ekluzek I forgot to wait for this mosart tag and already started aux_clm in #2838 pointing to mosart1.1.06 and rtm1_0_84. Should I stop and wait for this? I'm guessing not, but maybe I should wait before starting the mosart test-suite, yes?

Reply:
@slevis-lmwg actually ESCOMP/CTSM#2828 is using the older mosart/rtm tags since we are coordinating the same behavior updates with each of the three PR's. So you'll be using the existing mosart/rtm tags for those three, and we'll do another update after those to bring in the very latest mosart/rtm tags with the rpointer update. So your testing you've started is perfectly fine.

That's issue ESCOMP/CTSM#2923 to do the update to the rpointer mosart/rtm tags. That's something we can do on b4b-dev.